### DISCUSSION TITLE
Source mode problems

### DISCUSSION THREAD

**Discussion by Tamas-Toth-ebola** [cguZZman/plugin.googledrive#227] [Q&A] [answered]
Dear @cguZZman !

Am I right if I interpret that current latest _(v1.4.10 released Nov 12, 2020)_ version not works in 'source' mode? If yes could you please indicate it in the description of the plugin and on the WIKI page?

I'm so happy that you made this plugin, and very lot of thanks for it but I spent lot of time to search what is the problem on my side because of the non-working source plugin and just the end of the process found here at issues that there are more problems with it and if I'm right currently sadly not truly working.

Thanks for your answer and for your continuous efforts on the plugin. Life saver, absolutely!

**Comment by cguZZman** [comment, marked as answer]
Fair enough. It is now noted in the github description and will be updated in the wiki in the next release.
